hack/lib/cmd.sh

```diff
@@ -4,97 +4,97 @@
 
 # expect_success runs the cmd and expects an exit code of 0
 function os::cmd::expect_success() {
-	if [[ $# -ne 1 ]]; then echo "os::cmd::expect_success expects only one argument, got $#"; exit 1; fi
-	local cmd=$1
+    if [[ $# -ne 1 ]]; then echo "os::cmd::expect_success expects only one argument, got $#"; exit 1; fi
```

Comment:
I can't quite remember where the original problems were, but when editors pick up that tabs are used in a file they default `<tab>` key to be `'\t'` ... and then everyone begins using tabs for alignment instead of only indentation. Since Bash is often really *really* annoying to look at without the correct alignment of things, this was a PITA for me. I agree that the best approach is tabs for indentation and spaces for alignment ... but in practice it doesn't seem like that's possible.